client/src/App.js

```diff
@@ -27,29 +27,42 @@ class App extends Component {
   submitNewProject =  async e => {
     e.preventDefault();
     this.toggleForm();
-    
+
+    let title = e.target.querySelector('#form-title').value;
+    let email = e.target.querySelector('#form-email').value;
+    let link = e.target.querySelector('#form-link').value;
+    if (!title || !link || !email) {
+      // show error here
```

Comment:
lol
I like marking todos with `@todo:` to make them easier to search for/show up in the editor automatically. 😉